### PR TITLE
docs(tree): fix typo

### PR DESCRIPTION
**Pull request by yutao331763646**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
What problem this PR will resolve?

**Comment by afc163** [comment]
Closing for no response.

**Comment by yutao331763646** [comment]
> What problem this PR will resolve?

?

**Comment by yoyo837** [comment]
Feel free to open a new PR please.

**Comment by afc163** [comment]
What problem this PR will resolve?

**Comment by yoyo837** [comment]
Looks like a type improvement.

**Comment by yutao331763646** [comment]
> PR 这个会解决什么问题？
修改tree示例代码中的any类型为回调返回的实际类型，完善api文档中的类型

**Comment by afc163** [comment]
> 修改tree示例代码中的any类型为回调返回的实际类型

可以来个新 PR，把上面的内容区域都填一填。

File: components/tree/demo/basic.md [hunk 1]
```diff
@@ -47,11 +47,17 @@ const treeData = [
...
 
 const Demo = () => {
-  const onSelect = (selectedKeys: React.Key[], info: any) => {
+  const onSelect = (selectedKeys: React.Key[], info: {
+        event: 'select';
+        selected: boolean;
+        node: EventDataNode;
+        selectedNodes: DataNode[];
+        nativeEvent: MouseEvent;
+    }) => {
```
Comment (afc163) [inline components/tree/demo/basic.md:56]: 用户使用时应该不需要在参数里做这个定义吧，应该可以直接推断出来。

Comment (yutao331763646) [reply components/tree/demo/basic.md:56]: 如果不写，是能推断出来的，但是写了any ，就推断不了了

> 

好的

**Pull request closed without merging**